Anature/src/application/anatures/Anature.java

```diff
@@ -645,107 +401,57 @@ public BattleAnimationType getMoveAnimationType(int moveIndex)
 
 	boolean canCreate()
 	{
-		if(mName.isEmpty())
+		if(getName().isEmpty())
 		{
 			throw new IllegalStateException("The \"name\" variable was never set during construction.");
 		}
 
 		// TODO Is this a problem for wild anatures?
-		if(mOwnerName.isEmpty())
+		if(getOwner().isEmpty())
```

Comment:
It shouldn't be since wild anatures are supposed to be owned by "Wild". However we currently have it so all anature's are created with the player as the owner, which we should fix by having the owner's name be passed into the createAnature() method in AnatureBuilder.